src/components/autonomous/WaypointNav.js

```diff
@@ -27,18 +40,15 @@ function WaypointNav() {
   return (
   <form method="post" onSubmit={handleSubmit} className="waypoint-select"> 
     <div className="waypoint-select__params">
-    <label for="latitude">Latitude</label>
-    {submitted ? <input disabled type="number" name="latitude" placeholder="Latitude" /> : <input type="number" name="latitude" placeholder="Latitude" />}
-      <label for="longitude">Longitude</label>
-      {submitted ? <input disabled type="number" name="longitude" placeholder="Longitude" /> : <input type="number" name="longitude" placeholder="Longitude" />}
-      <div className="waypoint-checkbox">
-        <div>
-          <label>{submitted ? <input disabled type="checkbox" name="isApproximate" /> : <input type="checkbox" name="isApproximate" />} Approximate</label>
-        </div>
-        <div>
-          <label>{submitted ? <input disabled type="checkbox" name="isGate" />: <input type="checkbox" name="isGate" />} Is Gate</label>
-        </div>
-      </div>
+      <label htmlFor="latitude">Latitude</label>
+      {submitted ? <input disabled value={lat} onChange={e => e}/> : <input type="number" name="lat" value={lat} onChange={e => setLat(e.target.value)}/>}
+      <label htmlFor="longitude">Longitude</label>
+      {submitted ? <input disabled value={lon} onChange={e => e}/> : <input type="number" name="lon" value={lon} onChange={e => setLon(e.target.value)}/>}
```

Comment:
Changing the name of the input tags breaks the submit functionality. Take a look at `handleSubmit()`, the submitted form is converted to json and passed to the `requestWaypointNav()` method. It expects keys `latitude` and `longitude`, but it's getting `lat` and `lon`, so you should change the name of the inputs back.